qa/tasks/cephfs/test_volume_client.py

```diff
@@ -654,12 +650,8 @@ def test_get_authorized_ids(self):
             guest_entity_2=guest_entity_2,
         )))
         # Check the list of authorized IDs and their access levels.
-        if self.py_version == 'python3':
-            expected_result = [('guest1', 'rw'), ('guest2', 'r')]
-            self.assertCountEqual(str(expected_result), auths)
-        else:
-            expected_result = [(u'guest1', u'rw'), (u'guest2', u'r')]
-            self.assertItemsEqual(str(expected_result), auths)
+        expected_result = [('guest1', 'rw'), ('guest2', 'r')]
+        self.assertItemsEqual(str(expected_result), auths)
```

Comment:
what is this change for?

Reply:
Currently the test cases themselves are running and compatible with python2, and in the test cases it will run some python script with python3.

Reply:
IIUC, the whole point of this change is to drop python2 support? as the commit message reads

> The pybind now has dropped the WITH_PYTHON2 option, and for now only
> py3 supported.

if that's the case why bothering being python2 compatible?

> Currently the test cases themselves are running and compatible with python2

could you expand this a little bit? as you are practically reverting 21d6c5a95b4a6873bc8f35a74e3f48436bd96264, i added it to address test failure.

Reply:
For the local test, the vstart_runner.py as well as most tests in qa/ are only compatible with python2. Therefore, use python2 to run the tests.
More detail: locally.https://docs.ceph.com/docs/master/dev/developer_guide/running-tests-locally/

From the intergration test cases, I checked the logs, the ` test_volume_client ` test case itself is also running under python2(I add the code and get the python version from `sys.version_info.major`). And in the  ` test_volume_client ` test case it will run some inline python scripts by calling `run_python()`, which will try to import the `ceph_volume_client`, and  `ceph_volume_client` from `src/pybind/` only supports python3 as we discussed above.

Maybe I have missed some test case here ?
I added it back. By checking the version from `sys.version_info.major` instead of `self.py_version`.

Reply:
as discussed offline, the doc is out dated. fixed in #34636

Reply:
Cool. Thanks very much @tchaikov 
There still have some errors when running the local test cases under py3, which are none business with this PR here, and I will open another new PR to fix them all.